services/util/api_doc/util.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @apiDefine s_util Search > Util
+ * Information related to text of sections in the frontend
```

Comment:
This group is something more like "Utilities endpoints related with all other services", because the whole section will be for more than the texts

Reply:
Fixed!